bin/platinum.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+gcloud auth activate-service-account --key-file=/google-key.json
```

Comment:
this all seems very specific to the verification. ideally this container would be runnable by anyone. I think we should move this elsewhere.

Reply:
I actually did it this way so that we could keep them separate, my assumption was we would want Platinum to continue to be used the way it is today. But yeah I guess you're thinking rather than shipping the code we'd just share the Docker container with externals?

Let's discuss tomorrow.